### DISCUSSION TITLE
New feature for local media file streaming

### DISCUSSION THREAD

**Discussion by ercanhocaninyeri** [zauberzeug/nicegui#1026] [Q&A]
I started using Nicegui recently for developing an interface for my video processing stuff. I was a bit concerned about the lack of flexibility for playing local video files on the gui. Now, I feel very lucky to see this [new feature](https://github.com/zauberzeug/nicegui/pull/1016). 
I have tried it today and it seems to be running as expected. However, it took some time for me to figure out the codec requirements for the input video file. 
To be precise, if the codec of the input video is **MPEG-4 Visual**, it does not play the video at all. On the other hand, a video with **Advanced Video Codec (AVC)** works perfect. 
I am wondering if anyone has detailed information of the supported/unsupported codes or formats of the video files?

**Comment by rodja** [comment]
I also solved it by trying out multiple ffmpeg configurations and manual tests with the browsers I cared about. There very well be a table with video codec to web-browser compatibilities. But I have not found it :-(
